client/src/pages/EpisodeShowPage/Timer.tsx

```diff
@@ -0,0 +1,45 @@
+import * as React from "react";
+import * as moment from "moment";
+import { Glyphicon } from "react-bootstrap";
+
+export class Timer extends React.Component<any,  any> {
+  public render() {
+    return (
+      <span>
+        <span
+          style={{
+            display: this.props.timeLeft === Infinity ? "none" : "inline"
+          }}
+        >
+          <span
+            style={{
+              color: "#137a9a",
+              fontVariant: "small-caps",
+              fontWeight: 700,
+            }}
+          >
+            <Glyphicon glyph="time" /> time left
+          </span>
+            <br />
+          <span
+            style={{
+              fontSize: "28px",
+            }}
+          >
+            {moment(this.props.timeLeft).format("m")}
+          </span>
+          m
+          {" "}
+          <span
+            style={{
+              fontSize: "28px",
+            }}
+          >
+            {moment(this.props.timeLeft).format("ss")}
+          </span>
+          s
```

Comment:
If the goal is to have this formatted similar to "30m10s" I believe there is a way to do that with moment's formatting.

Reply:
Separately I've recently found out about Luxon, which may be better than moment for us (It may have trouble with older browsers by do not think that is a big deal)
https://moment.github.io/luxon/

Reply:
I briefly looked into Luxon. I was especially interested in whether it had more options for parsing strings into durations. But it didn't, and for our purposes seems close enough to moment. Advantages concerning time zones and i18n aren't really relevant for durations. Will keep in mind though.